lib/ui/pages/payment/payment_result.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:pispapp/controllers/flow/payment_flow_controller.dart';
+import 'package:pispapp/models/transaction.dart';
+import 'package:pispapp/routes/app_navigator.dart';
+import 'package:pispapp/ui/theme/light_theme.dart';
+import 'package:pispapp/ui/widgets/failure.dart';
+import 'package:pispapp/ui/widgets/success.dart';
+
+class PaymentResult extends StatelessWidget {
+  PaymentResult(this._paymentFlowController);
+
+  final PaymentFlowController _paymentFlowController;
+
+  @override
+  Widget build(BuildContext context) {
+    final transaction = _paymentFlowController.transaction;
+    final isSuccess = transaction.status == TransactionStatus.success;
+
+    // This screen is only intended to last for 3 seconds before it
+    // moves to the main dashboard.
+    // TODO(kkzeng): Shall we use a button instead of a timer like this?
```

Comment:
I think the timer is probably fine? We can worry about this later

Reply:
yup, sgtm